commits.go

```diff
@@ -399,7 +399,7 @@ type CommitStatus struct {
 	FinishedAt   *time.Time `json:"finished_at"`
 	Name         string     `json:"name"`
 	AllowFailure bool       `json:"allow_failure"`
-	Coverage     float64    `json:"coverage"`
+	Coverage     *float64   `json:"coverage"`
```

Comment:
Sorry but this is not a route I'm willing to go... A lot of these fields can be `null`, but I don't think this API will be usable if we have to `nil` check each and every field we want to use...